web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedbacklasthome.html

```diff
@@ -7,8 +7,15 @@
         <span data-ng-hide="row.published"> -
           <strong data-translate="">GUFwaitingForApproval</strong>
         </span>
+
+        <a class="btn btn-xs btn-link pull-right"
```

Comment:
I would remove `btn`, `btn-xs` and `btn-link` here, just the link with a red text is enough

Reply:
Done @MichelGabriel